pages/api/join.js

```diff
@@ -57,18 +57,8 @@ export default async function handler(req, res) {
     }
   }
 
-  await joinTable.create({
-    'Full Name': data.name,
-    'Email Address': data.email,
-    Student: data.educationLevel != 'tertiary' ? true : false,
-    Reason: data.reason,
-    Invited: open,
-    Club: data.club ? data.club : '',
-    IP: req.headers['x-forwarded-for'] || req.socket.remoteAddress
-  })
-
   if (open) {
-    let result = await postData(
+    postData(
```

Comment:
Unfortunately we have to await this– if we fire-and-forget and res.json, we might close this connection: https://docs-git-add-fire-and-forget-zeit.vercel.app/docs/v2/platform/limits?query=fire-and-forget#streaming-responses

Reply:
ugh no fun!!